docs/officers/payments/accept.rst

```diff
@@ -0,0 +1,46 @@
+:og:description: While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager. Treasurers also have the ability to apply waivers.
+
+Accepting Payments
+==================
+
+While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager.
+Treasurers also have the ability to apply waivers.
+
+.. hint::
+   To record cash or check payments, you must have a project manager or officer role.
+   To apply waivers, you need a separate permission.
+   If you need access, ask in :slack:`it-helpdesk`.
+
+Online-Only Payment Methods
+---------------------------
+
+The following payment methods are only accepted online, and incur an additional surcharge depending on the amount of the transaction.
+
+- Credit and debit cards
+- Google Pay
+- Apple Pay
+- Cash App
+
+We do **not** accept payments via PayPal, Venmo, or Zelle, as they're more complicated and expensive to process.
```

Comment:
```suggestion
We do **not** accept payments via PayPal, Venmo, or Zelle.
```

I don't feel like the "why" is useful to include here. 🤷

Reply:
I think it would be good to document _somewhere_. I started writing a separate admin page about payments but I don't have much to put in it.

Reply:
maybe bump it to a parenthetical then: `We do **not** accept payments via PayPal, Venmo, or Zelle (they're more complicated and expensive to process).`